src/scss/abstracts/_variables.scss

```diff
@@ -1,14 +1,13 @@
 /* Sass Variables */
 
 //Colors
-$black: #000000;
+$black: #000;
 $blue-grey: #474f78;
 $blue-grey-dark: #323755;
 $blue-grey-light: #adb0c4;
 $green: #0c5a45;
 $green-dark: #043326;
 $grey: #d8d8d8;
-$orange: #ffcc36;
+$orange: #ffb000;
```

Comment:
Why did you change the value of this color?

Reply:
That is the right color for orange.  Please see the attached screenshot
<img width="1201" alt="Screen Shot 2020-08-17 at 9 02 11 PM" src="https://user-images.githubusercontent.com/36742076/90461869-1268d880-e0cd-11ea-8bd6-bee29854f980.png">